scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -363,7 +386,7 @@ def cleanup(args):
         domain_cleanup(dom)
 
     for network in conn.listAllNetworks():
-        if network.name() == args.cloud + "-admin":
+        if network.name() in (args.cloud + "-admin", args.cloud + "-ironic"):
```

Comment:
why hard coding two types of networks, should be easier and more flexible to collect an array of networks that should be created, so the code does not care about the number and names